### ISSUE TITLE
Voice Processing Options

### ISSUE THREAD

**Issue by DoM1niC** [vector-im/element-call#554]
#### What would you like to do?
Voice Processing Settings 
#### Why would you like to do it?
Give User more Control over the Input Device
#### How would you like to achieve it?
Add four Options in Audio Settings Tab as a checkbox or better switch Option.... 
- Auto Gain Control
- Echo Canceling 
- Noise Canceling
- Voice Activation is already exist as PR https://github.com/vector-im/element-call/pull/492

**Comment by SimonBrandner** [comment]
Related: https://github.com/matrix-org/matrix-js-sdk/pull/2434, https://github.com/matrix-org/matrix-react-sdk/pull/8759

**Comment by DoM1niC** [comment]
> Related: [matrix-org/matrix-js-sdk#2434](https://github.com/matrix-org/matrix-js-sdk/pull/2434), [matrix-org/matrix-react-sdk#8759](https://github.com/matrix-org/matrix-react-sdk/pull/8759)

yes I already use this in my own Fork and works fine for simple 1:1 Calls.

**Comment by MrAnno** [comment]
matrix-org/matrix-js-sdk#2434 has been merged, making it possible to control advanced audio processing options.

AFAIK Element Call has its own "Settings UI", so the toggles implemented in matrix-org/matrix-react-sdk#8759 need to be added to Element Call separately.
